src/runners/handlers/slack.py

```diff
@@ -67,14 +68,18 @@ def handle(
     slack_token = vault.decrypt_if_encrypted(slack_token_ct)
 
     sc = SlackClient(slack_token)
-
+    client = WebClient(slack_token)
     # otherwise we will retrieve email from assignee and use it to identify Slack user
     # Slack user id will be assigned as a channel
 
     title = alert['TITLE']
 
     if recipient_email is not None:
-        result = sc.api_call("users.lookupByEmail", email=recipient_email)
+        if isinstance(recipient_email, str):
+            result = sc.api_call("users.lookupByEmail", email=recipient_email)
+
+        else:
+            result = client.conversations_open(users= recipient_email)
```

Comment:
looking at [the docs](https://api.slack.com/methods/conversations.open), it seems "conversation.open" takes a list of user id's, not user emails, so first this code needs to resolve a list of user emails to a list of user id's here

then below, you need to resolve swap out `sc.api_call("chat.postMessage", ...)` with `sc.api_call("conversations.open", ...)`